src/toil/jobStores/aws/jobStore.py

```diff
@@ -357,20 +397,24 @@ def _getKeyForUrl(url, existing=None):
         # Note that caller is responsible for closing the connection
         s3 = boto.s3.connect_to_region(region)
         try:
-            bucket = s3.get_bucket(url.netloc)
-            key = bucket.get_key(url.path[1:])
+            keyName = url.path[1:]
+            bucketName = url.netloc
+            bucket = s3.get_bucket(bucketName)
+            key = bucket.get_key(keyName)
             if existing is True:
                 if key is None:
-                    raise RuntimeError('Key does not exist.')
+                    raise RuntimeError("Key '%s' does not exist in bucket '%s'." %
```

Comment:
https://github.com/BD2KGenomics/toil/wiki/Python-Dos-and-Don'ts#dont-use-format-or--to-format-log-messages